Kitodo/src/main/webapp/WEB-INF/templates/includes/currentTasksEdit/taskBoxActivities.xhtml

```diff
@@ -123,6 +123,30 @@
                         <f:param name="zurueck" value="AktuelleSchritteBearbeiten"/>
                     </h:commandLink>
 
+                    <!-- Re-generate all images button -->
+                    <h:commandLink id="action12"
+                                   rendered="#{not empty task.typeGenerate and process.blockedUser == null}"
+                                   action="#{AktuelleSchritteForm.regenerateAllImagesButtonClick}"
```

Comment:
Form class `AktuelleSchritteForm` is renamed to `CurrentTaskForm` form. You should test your code after you solved merge conflicts. It would be better if you add Selenium tests for your added code.

Reply:
Thanks for the hint, Eclipse can not track that. Presumably that did not even lead to a merge conflict. Then I have to test it again, in fact.

You know, it's not that easy to test that because I need a whole setup for it. I need a process with pictures in it, if possible a broken one, and then I can configure everything (source folder, destination folder, ...) and then you can test the functionality, of course. And the next time I have to reload the database because of a change to the schema, the whole setup is gone. I also do not know how to solve this with Selenium tests, to be honest. Apart from the fact that they are also not running on my PC, probably a conflict with (unrelated) Visual Studio drivers, but I'm still not sure. What I mean is, even the Selenium test would need an appropriate setup. A process in the database whose ID and process title is known so that images can be copied to the right place, etc. Programming all this is probably more work than the development itself. Of course that would be nice, but my time is somehow finite.